openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/TableRepository.java

```diff
@@ -412,16 +412,21 @@ public Table addTableProfileData(UUID tableId, CreateTableProfile createTablePro
   public void deleteTableProfile(String fqn, String entityType, Long timestamp) {
     // Validate the request content
     String extension;
+    Class classMapper;
```

Comment:
@open-metadata/backend let me know if there is a better approach here as Sonar throws the following

> Generic types shouldn’t be used raw (without type parameters) in variable declarations or return values. Doing so bypasses generic type checking, and defers the catch of unsafe code to runtime.